mitiq/zne/tests/test_zne.py

```diff
@@ -540,3 +541,22 @@ def execute(circuit: qiskit.QuantumCircuit, shots: int = 8192) -> float:
     # Note: Unmitigated value is also (usually) within 10% of the true value.
     # This is more to test usage than effectiveness.
     assert abs(zne_value - true_value) < 0.1
+
+
+def test_execute_zne_transpiled_qiskit_circuit_with_noise_and_QFT():
```

Comment:
This is useful, but it's more like an "integration" test where we actually execute and test many components. Additional to this, it'd be helpful to have a unit test with a smaller scope, with the unit being the `from_qasm` function.

Reply:
I added another unit test to test the from_qasm() function! In doing so, I realized that it would be better to make the exception catch in the from_qasm() function as that function is more general and is called by the from_qiskit() function. Thank you for the comment!

Reply:
Hmmm this is tricky. Despite the function `from_qasm` is in the mitiq_qiskit module, its signature `def from_qasm(qasm: QASMType) -> cirq.Circuit` (as well as its current definition) makes it very generic and not necessarely related to qiskit. Admittedly the function could be moved elsewhere, perhaps in a mitiq_qasm module. In view of such a possible refactoring in the future, I would move your new code back to the `from_qiskit()` function, since it's specific to qiskit circuits. What do you think?

Reply:
With the most recent pull request, I moved the exception catch back to the from_qiskit() function. In doing so, I also had to change the test case to test the from_qiskit() function instead of the from_qasm() function. I agree that this makes more sense especially if some refactoring will be done. If anyone ever calls the from_qasm() function with a qasm string not from qiskit and **does not** want it to follow qiskit circuit formatting rules (for instance via mitiq!), then the way I had it would be very problematic. Unfortunately, this change means that if someone calls the from _qasm() function with a non-standard qiskit gate, it will throw an exception and won't go through the exception fallback which I believe is fine for the purpose of this issue since it wouldn't be from qiskit anymore?